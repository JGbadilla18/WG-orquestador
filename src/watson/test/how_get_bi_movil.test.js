const testWatson = require('./testWatson');

watsonText = 'Cómo solicito Bi Móvil'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
