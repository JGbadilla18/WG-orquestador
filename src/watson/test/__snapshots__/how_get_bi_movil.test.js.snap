// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cómo solicito Bi Móvil 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Conoce más aquí",
          "url": "https://blog.corporacionbi.com/solucionesbi/lp/solicitud-bi-movil",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Bi Móvil es el servicio que te mantiene alerta, notificándote sobre movimientos en cuentas, tarjetas o préstamos.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
